--- /linux-5.19.9/kernel/sys.c	2022-11-23 13:31:43.040003612
+++ /ker4/kernel/sys.c	2022-09-15 08:47:20.000000000 +0000
@@ -207,21 +207,6 @@
 	return error;
 }

-SYSCALL_DEFINE2(kernel_2d_memcpy, void *, dest, void *,src)
-{
-    float temp_matrix[2][2];
-    int ret1= __copy_from_user(temp_matrix, src, sizeof(temp_matrix));
-    if (ret1!=0){
-        printk("Error in copying from user space");
-    }
-    int ret2= __copy_to_user(dest, temp_matrix, sizeof(temp_matrix));
-    if (ret2!=0){
-        printk("Error in copying to user space");
-    }
-    return 0;
-}
-
-
 SYSCALL_DEFINE3(setpriority, int, which, int, who, int, niceval)
 {
 	struct task_struct *g, *p;
 	

